policyengine_canada/parameters/gov/provinces/sk/tax/income/credits/sk_senior_supplementary/amount.yaml

```diff
@@ -0,0 +1,17 @@
+description: Saskatchewan allows for this amount under the senior supplement credit.
+values:
+  2022-01-01: 1_336
+  2023-01-01: 1_421
+metadata:
+  unit: currency-CAD
+  label: Saskatchewan Senior Supplementary tax credit amount
+  reference:
+  - title: 2023 Saskatchewan TD1SK Personal Tax Credits Return, line 3
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1sk/td1sk-23e.pdf#page1
+  - title: 2023 Saskatchewan TD1SK-WS Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1sk-ws/td1sk-ws-23e.pdf
```

Comment:
add pdf page

Reply:
There is no disability amount policy being found in this legal pdf. So, I add a note: "Do not find the legal cite" below this reference.

Reply:
there should be one, they usually change the name a little bit maybe "physical impairment" etc